packages/connectivity/test/connectivity_test.dart

```diff
@@ -0,0 +1,77 @@
+import 'dart:async';
+import 'dart:typed_data';
+
+import 'package:flutter/services.dart';
+import 'package:connectivity/connectivity.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('$Connectivity', () {
```

Comment:
nit: is the `$` here on purpose?

Reply:
Yeah, the idea is to force you to update the string if you change the class name

Reply:
TIL! nice trick